docs/CODING_GUIDELINES_CORE.md

```diff
@@ -209,6 +209,19 @@ adhere to the following guidelines for our Core Web Component codebase.
   }
   ```
 
+  Content that is projected into a slot should not be styled by the host web
+  component. The component cannot make assumptions about the layout of the content
+  projected provided by the application. If the user needs certain layouts
+  defined within the projected content they can use layout utilities.
+
```

Comment:
`content provided by the application` not `content projected provided by the application`

also, i'm not sure what we mean by `should not be styled`. maybe a little more clarification there. it seems like it is talking about laying out the content. but the phrasing here sounds like any styling...